### DISCUSSION TITLE
Regarding contributing to the documentation

### DISCUSSION THREAD

**Discussion by carlwr** [Homebrew/discussions#841] [Everyday usage] [answered]
I see there is room for some improvements in the documentation ([FAQ](https://docs.brew.sh/FAQ), [Installation](https://docs.brew.sh/Installation#multiple-installations)) regarding having two separate Homebrew installs, especially as it applies to using Homebrew on the Arm/M1 macs. 

1. Is it recommended to discuss such changes at first, in broad terms, before a pull request? In that case, here at https://github.com/Homebrew/discussions or by creating an issue in the documentation repository?

2. Are there any guidelines for contributing to the documentation? (I'm aware of [Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct), [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md), mostly related to homebrew-core, and [Prose Style Guidelines](https://docs.brew.sh/Prose-Style-Guidelines)).

3. Are there any simple hints on how to suggest changes in the contets of the documentation? I've had a brief look at the [Homebrew/brew.sh](https://github.com/Homebrew/brew.sh) repository, but it's not immediately clear exactly what file to edit and then use for a pull request. Any hints are welcome, otherwise, I'll just put some more time into trying to understand the repo and Jekyll.

**Comment by carlocab** [comment]
Thanks for looking into improving the documentation and for reading up thoroughly on how before actually doing so. I'll try to get to your other questions later (or hopefully someone else will chime in before then), but for your third question:

I believe most (all?) of the content at https://brew.sh is generated from [here](https://github.com/Homebrew/brew/tree/master/docs), so this is actually where you'll want to open any PRs for changes.

**Reply by carlwr** [reply]
Thanks!

> I believe most (all?) of the content at https://brew.sh is generated from here, so this is actually where you'll want to open any PRs for changes.

That's really helpful!

**Comment by SMillerDev** [comment, marked as answer]
> Is it recommended to discuss such changes at first, in broad terms, before a pull request? In that case, here at https://github.com/Homebrew/discussions or by creating an issue in the documentation repository?

Nope, just make a pull request.

> Are there any guidelines for contributing to the documentation? (I'm aware of Code of Conduct, Contributing, mostly related to homebrew-core, and Prose Style Guidelines).

That's all I think

> Are there any simple hints on how to suggest changes in the contets of the documentation? I've had a brief look at the Homebrew/brew.sh repository, but it's not immediately clear exactly what file to edit and then use for a pull request. Any hints are welcome, otherwise, I'll just put some more time into trying to understand the repo and Jekyll.

The documentation you mentioned is in github.com/Homebrew/brew and then the docs folder.
